pyqt-apps/siriushla/li_di_bpms/bpm_main.py

```diff
@@ -1,18 +1,19 @@
-from re import S
+''' Diagnostic Interface of the LINAC's BPM'''
+
 import json
 from PyQt5.QtCore import Qt
-from qtpy.QtWidgets import QWidget, QLabel, QGridLayout, QGroupBox, QVBoxLayout, QTabWidget
-from pydm.widgets import PyDMLabel, PyDMLineEdit, enum_button, PyDMEnumComboBox
-from ..widgets import SiriusLedState
+from qtpy.QtWidgets import QGroupBox, QVBoxLayout, QTabWidget
+from qtpy.QtWidgets import QWidget, QLabel, QGridLayout
+from pydm.widgets import PyDMLabel, enum_button, PyDMEnumComboBox, PyDMSpinbox
 from siriushla.as_di_bpms.base import GraphWave
```

Comment:
leave an empty line between imports of external libraries and our own.
Also, you can import this class with relative import:
`from ..as_di_bpms.base import GraphWave`.